src/ros_interop/telemetry_manager.py

```diff
@@ -0,0 +1,25 @@
+import rospy
+from ros_interop.srv import TelemetrySrvResponse
+from ros_interop.msg import * 
+from geographic_msgs.msg import GeoPoint
+
+class TelemetryManager():
+
+    def __init__(self,interop_client) -> None:
+        self.interop_client = interop_client
+
+    def router(self,req):
```

Comment:
Add type hints for parameters and functions, parameter in this case would be TelemetrySrvRequest, function hint would be TelemetrySrvResponse